packages/library-sync/src/commonMain/kotlin/io/realm/mongodb/sync/SyncSession.kt

```diff
@@ -86,3 +114,13 @@ public interface SyncSession {
         public fun onError(session: SyncSession, error: SyncException)
     }
 }
+
+/**
+ * The possible states for [SyncSession] to be.
+ */
+public enum class SyncSessionState {
```

Comment:
There is a lot of documentation in Realm Java we can move over for these.

Reply:
I would probably move into`SyncSession.State`, similar to `User.State` and how it is done in Realm Java.